src/adql/translator/Q3CTranslator.java

```diff
@@ -153,35 +153,10 @@ public String translate(PolygonFunction polygon) throws TranslationException{
 		}
 	}
 
-	@Override
```

Comment:
Lots of deleted copypasta here from the first pass Q3CTranslator code, mea culpa.
Without the overrides, the parser will attempt generic postgres implementations, rather than incorrect pgSphere geometry grammar, when there is not an explicit Q3C use case.

Ultimately we probably want to clean this up in a fork&branch for an MR back to Gregory's base code for Q3C support, but not until it's well tested and possibly in production?